File: backend/src/Liturgy.Domain/Card.cs
namespace Liturgy.Domain;

public class Card
{
    public Guid Id { get; set; }
    public Guid ProjectId { get; set; }
    public Guid SprintId { get; set; }
    public string Code { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public Guid? AssigneeId { get; set; }
    public BoardColumn Column { get; set; } = BoardColumn.Backlog;
    public RKind? CurrentR { get; set; }
    public bool IsBlocked { get; set; }
    public DateTimeOffset CreatedAt { get; set; }
}

File: frontend/projects/liturgy/api/src/lib/models/card.ts
import { BoardColumn } from './board-column';
import { RKind } from './r-kind';

export interface Card {
  id: string;
  projectId: string;
  sprintId: string;
  code: string;
  title: string;
  assigneeId: string | null;
  assigneeInitials: string | null;
  column: BoardColumn;
  currentR: RKind | null;
  isBlocked: boolean;
  loggedCount: number;
}
